### PR TITLE
Fixed extracting of first sentence from the text containing backticks

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #128

### CODE REVIEW

File: models/BaseDoc.php [hunk 1]
```diff
@@ -40,6 +44,24 @@ class BaseDoc extends BaseObject
     public $tags = [];
 
 
+    /**
+     * @param Type|null $aggregatedType
+     * @return string[]|array
```
Comment (bizley) [inline models/BaseDoc.php:49]:
```suggestion
     * @return string[]
```

File: models/BaseDoc.php [hunk 2]
```diff
@@ -40,6 +44,24 @@ class BaseDoc extends BaseObject
...
+        if ($aggregatedType === null) {
+            return [];
+        }
+
+        $types = [];
+        foreach ($aggregatedType as $type) {
+            $types[] = (string) $type;
+        }
+
+        return $types ?: [(string) $aggregatedType];
```
Comment (bizley) [inline models/BaseDoc.php:62]: Is `Type`  stringable?

Comment (arogachev) [reply models/BaseDoc.php:62]: Yes, it is.

**Review by samdark — APPROVED** [review]

File: models/BaseDoc.php (hunk 1 above)
Comment (arogachev) [reply models/BaseDoc.php]: Thanks. Because PR is already merged, I'll fix it in a separate commit.

**Pull request merged**